shared/wallets/send-form/available/index.js

```diff
@@ -1,26 +1,38 @@
 // @flow
 import * as React from 'react'
 import * as Styles from '../../../styles'
-import {Box2, Text} from '../../../common-adapters'
+import * as Kb from '../../../common-adapters'
 
-type Props = {|
+export type Props = {|
   amountErrMsg: string,
 |}
 
-const Available = (props: Props) => (
-  <Box2 direction="vertical">
-    {!!props.amountErrMsg && (
-      <Text type="Body" style={styles.text}>
-        {props.amountErrMsg}
-      </Text>
-    )}
-  </Box2>
-)
+const Available = (props: Props) => {
+  // This will only work to apply one custom style.
```

Comment:
I couldn't find a way to tell `Markdown` that bold is supposed to become extrabold. I decided against extending it since we're moving to simple markdown.